client/src/components/Monument/Details/Address/Address.scss

```diff
@@ -1,6 +1,3 @@
-@import "../../../../../node_modules/bootstrap/scss/functions";
```

Comment:
This is unrelated to your PR, I just noticed I was importing these in some places where they don't need to be. I also have realized that these imports aren't getting deduplicated causing lots of duplicate bootstrap css... hopefully that's just in the dev version but if not we will need to figure out how to avoid this or else our compiled css size will blow up